tempesta.sh

```diff
@@ -2,20 +2,23 @@
 #
 # Tempesta FW service script.
 #
-# 2012-2014. Written by NatSys Lab. (info@natsys-lab.com).
+# Copyright (C) 2012-2014 NatSys Lab. (info@natsys-lab.com).
+# Copyright (C) 2015 Tempesta Technologies.
 
-SSOCKET=sync_socket
-TDB=tempesta_db
-TFW=tempesta_fw
-TFW_ROOT=`pwd`/$TFW
-TFW_CACHE_SIZE=`expr 256 \* 1024`
-TFW_CACHE_PATH=$TFW_ROOT/cache
+root=$(dirname "$0")
 
 arg=${1:-}
-ss_path=${SYNC_SOCKET:="./"}
-tdb_path=${TDB:="./"}
+ss_path=${SS_PATH:="$root/sync_socket"}
+tdb_path=${TDB_PATH:="$root/tempesta_db"}
+tfw_path=${TFW_PATH:="$root/tempesta_fw"}
+tfw_cfg_path=${TFW_CFG_PATH:="$root/tempesta_fw.conf"}
```

Comment:
Whoops, the default .conf file with comments was exactly there, but it seems I deleted it together with the old file.
Have to write it from the blank.